src/OpenTelemetry.ResourceDetectors.Process/OpenTelemetry.ResourceDetectors.Process.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry Extensions - Process Resource Detector.</Description>
+    <MinVerTagPrefix>ResourceDetectors.Process-</MinVerTagPrefix>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="1.6.0" />
```

Comment:
`OTelSdkVersion` can be used?

Reply:
From PR description:
> The plan is to release shortly as alpha with reference to OTel 1.6.0.
> Then create follow up help-wanted issue to extend it for another attributes and update OTel to 1.7.0.

Will do, just after initial release.